src/components/Tagline/Tagline.module.scss

```diff
@@ -0,0 +1,23 @@
+.tagline {
+  padding: 30px 20px;
+  width: 100vw;
+  display: flex;
+  flex-direction: column;
+}
+
+.taglinePara {
```

Comment:
typically we dont allow any shorthand. so `taglineParagraph`. also there's a feature in vscode to find/replace all occurrences. text me if u dont know about it